src/openms/source/ANALYSIS/ID/SimpleSearchEngineAlgorithm.cpp

```diff
@@ -69,6 +76,7 @@ namespace OpenMS
     IntList isotopes = {0, 1};
     defaults_.setValue("precursor:isotopes", isotopes, "Corrects for mono-isotopic peak misassignments. (E.g.: 1 = prec. may be misassigned to first isotopic peak)");
 
+    // Setting fragment mass tolerance and its unit
```

Comment:
In my view, slightly better than having multiple comments above: here, the comment will actually save reading time when skimming through the lines below